src/band/diag_pseudo_potential.cpp

```diff
@@ -251,442 +242,442 @@ Band::diag_pseudo_potential_exact(int ispn__, Hamiltonian_k<real_type<T>>& Hk__)
-
-            if (ctx_.cfg().control().verification() >= 1) {
-                double max_diff = check_hermitian(hmlt, N + expand_with - num_locked);
-                if (max_diff > 1e-12) {
-                    std::stringstream s;
-                    kp.message(1, __function_name__, "H matrix of size %i is not Hermitian, maximum error: %18.12e\n",
-                               N + expand_with - num_locked, max_diff);
-                }
-            }
-
-            /* increase size of the variation space */
-            N += expand_with;
-
-            // Copy the Ritz values
-            eval >> eval_old;
-
-            /* solve standard eigen-value problem with the size N - num_locked. */
-            kp.message(3, __function_name__, "Computing %d pre-Ritz pairs\n", num_bands - num_locked);
-            if (std_solver.solve(N - num_locked, num_bands - num_locked, hmlt, &eval[0], evec)) {
-                std::stringstream s;
-                s << "error in diagonalization";
-                TERMINATE(s);
-            }
-
-            ctx_.evp_work_count(std::pow(static_cast<double>(N - num_locked) / num_bands, 3));
-
-            kp.message(3, __function_name__, "step: %i, current subspace size: %i, maximum subspace size: %i\n", k, N, num_phi);
-            for (int i = 0; i < num_bands - num_locked; i++) {
-                kp.message(4, __function_name__, "eval[%i]=%20.16f, diff=%20.16f, occ=%20.16f\n", i, eval[i],
-                    std::abs(eval[i] - eval_old[i]), kp.band_occupancy(i, ispin_step));
-            }
-            niter++;
-        }
-    } /* loop over ispin_step */
-    PROFILE_STOP("sirius::Band::diag_pseudo_potential_davidson|iter");
-
-    //if (ctx_.print_checksum()) {
-    //    auto cs = psi.checksum(0, ctx_.num_fv_states());
-    //    if (kp__->comm().rank() == 0) {
-    //        DUMP("checksum(psi): %18.10f %18.10f", cs.real(), cs.imag());
-    //    }
-    //}
-    if (is_device_memory(ctx_.preferred_memory_t())) {
-        for (int ispn = 0; ispn < ctx_.num_spins(); ispn++) {
-            psi.pw_coeffs(ispn).copy_to(memory_t::host, 0, num_bands);
-            psi.pw_coeffs(ispn).deallocate(memory_t::device);
-        }
-    }
-
-    kp.release_hubbard_orbitals_on_device();
-    //== std::cout << "checking psi" << std::endl;
-    //== for (int i = 0; i < ctx_.num_bands(); i++) {
-    //==     for (int j = 0; j < ctx_.num_bands(); j++) {
-    //==         double_complex z(0, 0);
-    //==         for (int ig = 0; ig < kp__->num_gkvec(); ig++) {
-    //==             z += std::conj(psi.pw_coeffs(0).prime(ig, i)) * psi.pw_coeffs(0).prime(ig, j);
-    //==         }
-    //==         if (i == j) {
-    //==             z -= 1.0;
-    //==         }
-    //==         if (std::abs(z) > 1e-10) {
-    //==             std::cout << "non-orthogonal  wave-functions " << i << " " << j << ", diff: " << z << std::endl;
-    //==         }
-    //==     }
-    //== }
-
-    return niter;
-}
+//template <typename T>
```

Comment:
Can you just delete this instead of commenting it?

Reply:
It was just here so that we can easily revert back if using the Davidson stand-alone function does not pass the CI test

Reply:
Ah okay, but there is also `git revert` :upside_down_face: